src/Main.java

```diff
@@ -1,5 +1,47 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задача№1");
+        int[] payments = {191000, 470000, 199000, 220000, 370000};
+        int sum = 0;
+        for (int i = 0; i < payments.length; i++) {
+            sum = sum + payments[i];
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+
+        System.out.println();
+        System.out.println("Задача№2");
+        int maxPayments = 0;
+        for (int i = 0; i < payments.length; i++) {
+            if (payments[i] > maxPayments) {
+                maxPayments = payments[i];
+            }
+        }
+        System.out.println("Максимальная сумма трат за неделю составила " + maxPayments + " рублей");
+        int minPayments = payments[0] + 1;
```

Comment:
А зачем +1? Можно оставить просто
```suggestion
        int minPayments = payments[0];
```